### PR TITLE
Support wide angle camera

### PR DESCRIPTION
**Pull request by iche033**
# 🎉 New feature

## Summary

Updates the Sensor DOM to support parsing wide angle cameras.

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

**Review by ahcorde — DISMISSED** [review]
can you add some tests?

**Comment by iche033** [comment]
added integration test in 2800f2c

**Review by scpeters — APPROVED** [review]

File: src/Sensor.cc [hunk 1]
```diff
@@ -64,7 +64,8 @@ const std::vector<std::string> sensorTypeStrs =
   "navsat",
   "segmentation_camera",
   "boundingbox_camera",
-  "custom"
+  "custom",
+  "wideanglecamera"
```
Comment (chapulina) [inline src/Sensor.cc:68]: We should document this string here: https://github.com/ignitionrobotics/sdformat/blob/f2a286cbf89aca7daa722a70a357e0de32224372/sdf/1.9/sensor.sdf#L10-L36

Most other cameras are using `camel_case` (at least partially), can we do it here too? i.e. `wideangle_camera` or `wide_angle_camera`, or all of them to be safe?

Comment (scpeters) [reply src/Sensor.cc:68]: the existing convention with gazebo classic is [wideanglecamera](https://github.com/osrf/gazebo/blob/gazebo11/worlds/lensflare_plugin.world#L97). I think we can choose to add alternates, but I would keep `wideanglecamera` for backwards compatibility

Comment (iche033) [reply src/Sensor.cc]: changed `wideanglecamera` to `wide_angle_camera` here. Also updated the Load function to check for both `wide_angle_came` and `wideanglecamera` when loading the sdf for backward compatibility. 
8aacb98

**Review by ahcorde — APPROVED** [review]

**Comment by MrKeith99** [comment]
Is there any possibility to add this feature in `Gazebo Fortress` as well?
I am struggling to use `Humble` and `gz_ros2_control` within the `Gazebo Fortress` or `Gazebo Garden`.

**Comment by iche033** [comment]
the PRs to add wide angle camera support in gz-rendering (https://github.com/gazebosim/gz-rendering/pull/490 and https://github.com/gazebosim/gz-rendering/pull/733) include ABI breaking changes so I think it'll be difficult to backport all wide angle camera related changes to Fortress.

**Pull request merged**